Composer/packages/client/src/components/ProjectTree/ProjectTree.tsx

```diff
@@ -257,36 +313,53 @@ export const ProjectTree: React.FC<Props> = ({
           icon={bot.isRemote ? icons.EXTERNAL_SKILL : icons.BOT}
           isActive={doesLinkMatch(link, selectedLink)}
           link={link}
-          menu={[{ label: formatMessage('Create/edit skill manifest'), onClick: () => {} }]}
+          menu={menu}
           onSelect={handleOnSelect}
         />
       </span>
     );
   };
 
   const renderDialogHeader = (skillId: string, dialog: DialogInfo) => {
-    const warningContent = notificationMap[skillId][dialog.id]
-      ?.filter((diag) => diag.severity === DiagnosticSeverity.Warning)
-      .map((diag) => diag.message)
-      .join(',');
-    const errorContent = notificationMap[skillId][dialog.id]
-      ?.filter((diag) => diag.severity === DiagnosticSeverity.Error)
-      .map((diag) => diag.message)
-      .join(',');
-
+    const diagnostics: Diagnostic[] = notificationMap[rootProjectId][dialog.id];
     const link: TreeLink = {
       dialogId: dialog.id,
       displayName: dialog.displayName,
       isRoot: dialog.isRoot,
+      diagnostics,
       projectId: rootProjectId,
       skillId: skillId,
-      errorContent,
-      warningContent,
     };
+    const menu: any[] = [
+      {
+        label: formatMessage('Add a trigger'),
+        icon: 'Add',
+        onClick: () => {
+          onDialogCreateTrigger(skillId, dialog.id);
+        },
+      },
+      {
```

Comment:
Why is there an empty item?

Reply:
It's the divider item in `treeItem.tsx`.